spec/myapp_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'spec_helper'
+require_relative '../app/models/shop'
+
+describe 'myapp' do
+  include Rack::Test::Methods
+
+  def app
+    Sinatra::Application.new
+  end
+
+  describe 'root path' do
+    before do
+      @shop = Shop.new({name: 'mandy', description: 'bla'})
+      @shop.id = @shop.create
+      get '/v1/shops'
+    end
+
+    after do
+      @shop.delete
```

Comment:
there is no need to delete the shop record here, as `database cleaner` is configured already.